exercises/webapp/src/components/HackerNews.spec.js

```diff
@@ -0,0 +1,68 @@
+import {
+  mount
+} from '@vue/test-utils';
+import HackerNews from './HackerNews.vue';
+import NewsItem from './NewsItem.vue';
+
+const createWrapper = (options) => {
+  const defaults = {
+    propsData: {
+      initialNews: [{
+        id: 0,
+        title: 'Message 1',
+        voteCount: 2
+      }, {
+        id: 1,
+        title: 'Message 2',
+        voteCount: 4
+      }],
+    }
+  };
+  return mount(HackerNews, { ...defaults, ...options });
+};
+
+describe('Sorting Order', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = createWrapper({});
+  });
+
+  it('sorting order', async () => {
```

Comment:
"It sorting order"? This reads like broken English in my opinion :wink:

What about:
```js
describe('click switch order button', () => {
  it('toggles ascending/descending order', async () => {
    /* ... */
  })
})
```